functions/.env

```diff
@@ -1 +1,2 @@
 TYPESENSE_API_URL=https://o89yhjf824.execute-api.us-east-1.amazonaws.com/search
+FUNCTIONS_API_BASE=https://us-central1-digital-testimony-prod.cloudfunctions.net
```

Comment:
This default is for the deployed DEV environment - this should be the link to the `digital-testimony-dev` base and I think the prod env vars can go in the `.env.digital-testimony-prod` file.

I think `.env.demo-dtp` handles the defaults when running locally with `yarn dev:up` (though that will need to be ngroked, so I wouldn't stress about that one.